README.md

```diff
@@ -7,6 +7,9 @@ extends support for the Bluespec Extensible RISC implementation (BERI).
 To build and run CheriBSD we recommend using the
```

Comment:
We should probably update this paragraph to be much more general, given we have both emulators and non-MIPS architectures, but that's a separate concern from this PR.